packages/fcl-core/src/interaction-template-utils/derive-cadence-by-network/derive-cadence-by-network.test.js

```diff
@@ -0,0 +1,224 @@
+                    pin_contract_name: "FungibleToken",
+                    pin_contract_address: "0x9a0766d93b6608b7",
+                    imports: [],
+                  },
+                },
+                {
+                  network: "emulator",
+                  address: "0xee82856bf20e2aa6",
+                  dependency_pin_block_height: 0,
+                },
+              ],
+            },
+          ],
+        },
+      ],
+      parameters: [
+        {
+          label: "amount",
+          index: 0,
+          type: "UFix64",
+          messages: [
+            {
+              key: "title",
+              i18n: [
+                {
+                  tag: "en-US",
+                  translation: "Amount",
+                },
+              ],
+            },
+            {
+              key: "description",
+              i18n: [
+                {
+                  tag: "en-US",
+                  translation: "The amount of FLOW tokens to send",
+                },
+              ],
+            },
+          ],
+        },
+        {
+          label: "to",
+          index: 1,
+          type: "Address",
+          messages: [
+            {
+              key: "title",
+              i18n: [
+                {
+                  tag: "en-US",
+                  translation: "To",
+                },
+              ],
+            },
+            {
+              key: "description",
+              i18n: [
+                {
+                  tag: "en-US",
+                  translation: "The Flow account the tokens will go to",
+                },
+              ],
+            },
+          ],
+        },
+      ],
+    },
+  }
+
+  test("v1.1.0, It derives cadence correctly for a given network", async () => {
+    let cadence = await deriveCadenceByNetwork({
+      network: "mainnet",
+      template: template11,
+    })
+
+    const expectedCadence = `import FungibleToken from 0xf233dcee88fe0abe\n\n#interaction(\n    version: "1.1.0",\n    title: "Transfer Flow",\n    description: "Transfer Flow to account",\n    language: "en-US",\n    parameters: [\n        Parameter(\n            name: "amount", \n            title: "Amount", \n            description: "The amount of FLOW tokens to send"\n        ),\n        Parameter(\n            name: "to", \n            title: "To",\n            description: "The Flow account the tokens will go to"\n        )\n    ],\n)\n\ntransaction(amount: UFix64, to: Address) {\n    let vault: @FungibleToken.Vault\n    \n    prepare(signer: AuthAccount) {\n        self.vault \u003c- signer\n            .borrow\u003c\u0026{FungibleToken.Provider}\u003e(from: /storage/flowTokenVault)!\n            .withdraw(amount: amount)\n    }\n\n    execute {\n        getAccount(to)\n            .getCapability(/public/flowTokenReceiver)!\n            .borrow\u003c\u0026{FungibleToken.Receiver}\u003e()!\n            .deposit(from: \u003c-self.vault)\n    }\n}`
```

Comment:
is there any way to format it in a more readable way? probably even use `.replace` for `\n`?